src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -140,6 +156,7 @@ export default class DefaultBrowserBehavior implements BrowserBehavior {
     return this.isChrome() || this.isEdge();
   }
 
+  // TODO: Deprecated, needs to be removed
```

Comment:
We probably wants to at least to do like console.warn and say this is no longer supported and just return false all the time. This API is already wrong anyway.